CMakeLists.txt

```diff
@@ -33,13 +63,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 ")
 include_directories(
   include
   ${catkin_INCLUDE_DIRS}
-  ${OPENVR}/headers
```

Comment:
may be you do not need to remove existing lines, unless that's the cause of any trouble